### DISCUSSION TITLE
Onboarding PR in main branch whereas renovate.json exist on baseBranches

### DISCUSSION THREAD

**Discussion by clementfumey** [renovatebot/renovate#24563] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Docker renovate/renovate:36-full

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I am using renovate from Jenkins and Docker with an environnement variable set to `RENOVATE_BASE_BRANCHES = 'test-renovate' `.
In my project branch `test-renovate` I have set a renovate.json file. When launching renovate I am expecting it to create a PR from renovate/test-renovate-myoutdatedpackage-x.x to test-renovate . Instead it creates an onboarding PR from 
renovate/configure to master . There is no renovate.json file in master branch, it is present only in test-renovate and ou intergration process need me to proove that it works first in my-branch before merging it to master.

Am I doing something wrong here ? Can I force renovate to ignore onboarding ?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
Renovate requires config to be in the default branch

**Reply by rarkins** [reply, marked as answer]
You can set onboarding=false and requireConfig=false
